db/seeds/support/location_seeder.rb

```diff
@@ -19,7 +19,8 @@ class LocationSeeder
   FILE = Rails.root.join("db", "seeds", "support", "locations.csv")
   SEPARATOR = ";"
   ENCODING = "UTF-8"
-
+  SEED_MARKER = "locations seeded"
```

Comment:
Wird der noch irgendwo verwendet? Ich dachte, dass das eher nur intern verwendet wird. Und mit den Anpassungen in diesem Branch sollten eher auch die externen Refrenzen auf diese Konstante entfernt werden. Dachte ich. 

Übersehe ich einen Use-Case?